utils/LocalHelpers.ts

```diff
@@ -0,0 +1,24 @@
+import type { HardhatRuntimeEnvironment } from "hardhat/types";
+import type { ERC20 } from "../typechain/ERC20";
+import * as constants from "../utils/Constants";
+import { numberToBigNumberFixed, to_d18 } from "./NumbersHelpers";
+
+export async function getUsdcFor(hre: HardhatRuntimeEnvironment, receiverAddress: string, amount: number) {
```

Comment:
I don't get this function what do we use it for?
both its name and usage are not clear